assets/package.json

```diff
@@ -14,13 +14,13 @@
     "@babel/preset-env": "^7.0.0",
     "babel-loader": "^8.0.0",
     "codemirror": "^5.49.2",
-    "copy-webpack-plugin": "^4.5.0",
+    "copy-webpack-plugin": "^5.1.1",
```

Comment:
I think we had issues with the latest version of webpack when we were first trying to setup stuff and we decided not to update them until we figured out what was wrong. Was that for this branch or when we were trying out monaco?

Reply:
i dont remember